### ISSUE TITLE
Integration in a FlaskApp in /admin

### ISSUE THREAD

**Issue by gferreux** [syrusakbary/Flask-SuperAdmin#135]
Is it possible to integrate Flask-SuperAdmin in an exiting FlaskApp on a specific route '/admin' without running 2 app.run ? Thanks

**Comment by wojcikstefan** [comment]
Yes, it is. You simply need to initialize your admin this way:  `admin = Admin(app=app, url='/admin')` where `app` is your FlaskApp.
